contrib/cidmap/Makefile.am

```diff
@@ -1,5 +1,59 @@
-dist_pkgdata_DATA = Adobe-CNS1-6.cidmap Adobe-Identity-0.cidmap		\
-	Adobe-Japan1-6.cidmap Adobe-Korea1-2.cidmap Adobe-GB1-5.cidmap	\
-	Adobe-Japan1-5.cidmap Adobe-Japan2-0.cidmap
+# Copyright (C) 2018 Jose Da Silva
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see http://www.gnu.org/licenses/.
 
--include $(top_srcdir)/git.mk
```

Comment:
Yeah you shouldn't have removed these, and all your new makefiles needs it too.

```
$git status
On branch master
Untracked files:
  (use "git add <file>..." to include in what will be committed)

        contrib/Makefile
        contrib/Makefile.in
        contrib/admintools/Makefile
        contrib/admintools/Makefile.in
        contrib/cidmap/Makefile
        contrib/cidmap/Makefile.in
        contrib/cidmap/src/.deps/
        contrib/cidmap/src/.libs/
        contrib/cidmap/src/Makefile
        contrib/cidmap/src/Makefile.in
        contrib/cidmap/src/mapcns1.exe
        contrib/cidmap/src/mapcns1.o
        contrib/cidmap/src/mapgb1.exe
        contrib/cidmap/src/mapgb1.o
        contrib/cidmap/src/mapjapan1.exe
        contrib/cidmap/src/mapjapan1.o
        contrib/cidmap/src/mapjapan2.exe
        contrib/cidmap/src/mapjapan2.o
        contrib/cidmap/src/mapkorean.exe
        contrib/cidmap/src/mapkorean.o
        contrib/fonttools/.deps/
        contrib/fonttools/.libs/
        contrib/fonttools/Makefile
        contrib/fonttools/Makefile.in
        contrib/fonttools/acorn2sfd-acorn2sfd.o
        contrib/fonttools/acorn2sfd.exe
        contrib/fonttools/dewoff.exe
        contrib/fonttools/dewoff.o
        contrib/fonttools/findtable.exe
        contrib/fonttools/findtable.o
        contrib/fonttools/pcl2ttf.exe
        contrib/fonttools/pcl2ttf.o
        contrib/fonttools/pfadecrypt.exe
        contrib/fonttools/pfadecrypt.o
        contrib/fonttools/rmligamarks.exe
        contrib/fonttools/rmligamarks.o
        contrib/fonttools/showttf.exe
        contrib/fonttools/showttf.o
        contrib/fonttools/stripttc.exe
        contrib/fonttools/stripttc.o
        contrib/fonttools/ttf2eps-ttf2eps.o
        contrib/fonttools/ttf2eps.exe
        contrib/fonttools/woff.exe
        contrib/fonttools/woff.o
```

Reply:
then we add it in in a patch.

Reply:
Did you compile this locally at all? It would have been fairly easy to spot this...

Reply:
Compiled locally (in a cloned directory, then files transferred back to master)
...then compiled again on the pull request - this came up all checkmarks good.
never dirtied-up the master, so never had to run that command - new to me.